From a6e599e79bd15f6a97ff6d8f2e3109759ff37aaf Mon Sep 17 00:00:00 2001
From: Thorsten Kukuk <kukuk@thkukuk.de>
Date: Mon, 13 Nov 2017 11:15:21 +0100
Subject: [PATCH] Include sys/param.h instead of rpc/types.h for
 MAXHOSTNAMELEN, we are not using RPC functions and sunrpc is deprecated in
 glibc.

---
 src/YQUI.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/YQUI.cc b/src/YQUI.cc
index 7c82468..f949979 100644
--- a/src/YQUI.cc
+++ b/src/YQUI.cc
@@ -22,7 +22,7 @@
 
 /-*/
 
-#include <rpc/types.h>		// MAXHOSTNAMELEN
+#include <sys/param.h>		// MAXHOSTNAMELEN
 #include <dlfcn.h>
 #include <libintl.h>
 #include <algorithm>
